doc/source/conf.py

```diff
@@ -129,14 +151,19 @@
 # html_theme_options = {}
 html_theme_options = {
     'logo': 'logo.png',
-    'github_user': 'ansible',
-    'github_repo': 'molecule',
+    'github_user': github_repo_org,
+    'github_repo': github_repo_name,
     'github_button': True,
+    'github_banner': True,
+    'github_type': 'star',
+    'github_count': True,
     'badge_branch': 'master',
     'travis_button': True,
+    'codecov_button': True,
+    # 'analytics_id': ...,
```

Comment:
I have this set in the `Advanced Settings` in RTD project settings, not sure if you'd want it here as well
```suggestion
     'analytics_id': 'https://github.com/ansible/community/issues/391',
```

Reply:
@gundalow it looks like you've copy-pasted the wrong thing :)

P.S. I usually prefer file-based configs as Git tracks them better throughout the history + everything is in one place.

Reply:
Urgh, need coffee

Reply:
https://molecule.readthedocs.io/en/latest/_static/readthedocs-data.js has
```js
var READTHEDOCS_DATA = {
    [...]
    global_analytics_code: 'UA-17997319-1',
    user_analytics_code: 'UA-128382387-1'
};
```
I guess it's the last one? So it'd be:
```suggestion
    'analytics_id': 'UA-128382387-1',
```

Reply:
Yup, `UA-128382387-1` is correct.